src/shared/vendor/third_party/webcomponents.js

```diff
@@ -6374,4 +6374,6 @@ CustomElements.addModule(function(scope) {
   window.Platform = scope;
 })(window.WebComponents);
 
-module.exports = window.WebComponents;
+if (typeof exports !== 'undefined') {
```

Comment:
For whatever reason, this seems to be the standard check for a UMD wrapper:

https://github.com/umdjs/umd/blob/81c95124cbdcb3fce883418ba2882d9cd7d9b2c4/returnExports.js#L21